include/player.h

```diff
@@ -1,15 +1,20 @@
+#pragma once
+
+#include "types.h"
 #include <string>
 #include <vector>
-#include "types.h"
 
 // Player object
 namespace BitBorn {
+
+enum Action { MOVE_FORWARD, MOVE_BACK, MOVE_LEFT, MOVE_RIGHT, LOOK_LEFT, LOOK_RIGHT };
```

Comment:
`>_>`

Reply:
Is that a good `>_>` or a bad `>->`? This was done by `clang-format` not me.